.github/workflows/generate-release-candidate.yml

```diff
@@ -0,0 +1,29 @@
+name: Generate Release Candidate
+
+on: 
+  repository_dispatch:
+    types: [generate-release-candidate]
+
+jobs:
+  Generate-Release-Candidate:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_STAGING_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_STAGING_SECRET_ACCESS_KEY }}
+          aws-region: us-west-2
+
+      - name: Install python packages
+        run: |
+          sudo pip install boto3
+          sudo pip install pyyaml
+          sudo pip install requests
+    
+      - name: Moving artifacts from snapshots to release folder
+        run: |
+          cd release-tools
+          ./generate-release-candidate.py ${GITHUB_RUN_ID}
```

Comment:
Do you need to test if the server has python 3.7 or not?

Reply:
Not needed. As this will only be run on workflow where the python is always > 3.7. Added this information in case if we ever want to run it from the desktop.

Reply:
You shouldnt think that way, considering this can be part of the Anytime Builds later.
You should consider also running it locally.
However, you did mention the requirements in comments, so I think ok.